utility/src/main/java/org/oppia/android/util/accessibility/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+kt_android_library(
+    name = "accessibility_module",
+    srcs = ["AccessibilityModule.kt"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":android_accessibility_manager",
+        ":custom_accessibility_manager",
+        artifact("com.google.dagger:dagger:2.27"),
+        artifact("javax.inject:javax.inject:1"),
+    ],
+)
+
+kt_android_library(
+    name = "android_accessibility_manager",
+    srcs = ["AndroidAccessibilityManager.kt"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":custom_accessibility_manager",
+        artifact("javax.inject:javax.inject:1"),
+    ],
+)
+
+kt_android_library(
+    name = "custom_accessibility_manager",
```

Comment:
This is the API, and the other two are the implementations for this package. I think we should instead set this up as:

```bazel
kt_android_library(
    name = "accessibility",
    visibility = ["//visibility:public"],
    srcs = ["CustomAccessibilityManager.kt"],
    deps = [":impl"],
)

kt_android_library(
    name = "impl",
    srcs = ["AndroidAccessibilityModule.kt","AccessibilityModule.kt"],
    deps = [...],
)

kt_android_library(
    name = "testing", # Could also be called 'fake_accessibility_manager'
    visibility = ["//visibility:public"],
    srcs = ["FakeAccessibilityManager.kt"],
    deps = [...],
)
```

This is the convention we should be using broadly since it clearly designates the public API as the package-level library, and properly privatizes the implementation & separates out the testing library.

For packages that have multiple public APIs (e.g. general utility packages with a bunch of separate things), in that case we would not have a package-level library and each group of utilities would be in their own packages.

@fsharpasharp FYI since we discussed this today.

Reply:
Done, instead of `accessibility` I've used `acc` as it was getting refactored in app/BUILD.bazel. Will change it back after conformation.